.github/workflows/build_and_run_example_test.yml

```diff
@@ -0,0 +1,90 @@
+name: Build Websockets
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        idf_ver: ["latest"]
+        idf_target: ["esp32"]
+
+    runs-on: ubuntu-20.04
+    container: espressif/idf:${{ matrix.idf_ver }}
+    steps:
+      - name: Checkout esp-protocols
+        uses: actions/checkout@master
+        with:
+          path: esp-protocols
+      - name: Build ${{ matrix.example }} with IDF-${{ matrix.idf_ver }} for ${{ matrix.idf_target }}
+        env:
+          IDF_TARGET: ${{ matrix.idf_target }}
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          cd $GITHUB_WORKSPACE/esp-protocols/components/esp_websocket_client/examples/
+          idf.py build
+      - name: Merge binaries
+        env:
+          IDF_TARGET: ${{ matrix.idf_target }}
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          cd $GITHUB_WORKSPACE/esp-protocols/components/esp_websocket_client/examples/build
+          esptool.py --chip ${{ matrix.idf_target }} merge_bin --fill-flash-size 4MB -o flash_image.bin @flash_args
+      - uses: actions/upload-artifact@v2
+        with:
+          name: examples_app_bin_${{ matrix.idf_target }}_${{ matrix.idf_ver }}
+          path: esp-protocols/components/esp_websocket_client/examples/build/*
+
+  run-target:
+    name: Run Example Test on target
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        idf_ver: ["latest"]
+        idf_target: ["esp32"]
+    runs-on:
+      - ESP32-ETHERNET-KIT
+    container:
+      image: python:3.7-buster
+      options: --privileged # Privileged mode has access to serial ports
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v2
+        with:
+          name: examples_app_bin_${{ matrix.idf_target }}_${{ matrix.idf_ver }}
+          path: examples/build
+      - name: Install Python packages
+        env:
+          PIP_EXTRA_INDEX_URL: "https://www.piwheels.org/simple"
+        run: | 
+          pip install esptool pytest-embedded==0.6.2 pytest-embedded-serial-esp==0.6.2
+          pip install junit_xml SimpleWebSocketServer pyserial pytest-embedded-serial pytest-embedded-serial-esp
```

Comment:
Hi, seems like here you missed package `pytest-embedded-idf`

suggest install by

```shell
pip install pytest-embedded-serial-esp~=0.6 pytest-embedded-idf~=0.6
pip install junit_xml SimpleWebSocketServer
```
`esptool` and `pyserial` would be installed by `pytest-embedded-serial-esp` automatically.

Reply:
Hi, thanks for the suggestions. I updated it, but still got the same error
```
=================================== FAILURES ===================================
_______________________ test_examples_protocol_websocket _______________________
pytest_websocket.py:110: in test_examples_protocol_websocket
    if 'WEBSOCKET_URI_FROM_STDIN' in dut.app.sdkconfig:
E   AttributeError: 'App' object has no attribute 'sdkconfig'
------------------------------ Captured log setup ------------------------------
INFO     root:serial.py:68 Target: esp32, Port: /dev/ttyUSB1
INFO     root:dut.py:33 Logs recorded under folder: /tmp/pytest-embedded/2022-04-06_07-55-02/test_examples_protocol_websocket
```